install/scaling.md

```diff
@@ -12,3 +12,34 @@ Size of the cluster is controlled by a `size` key in the Custom Resource options
 In this example we have changed the size of the Percona XtraDB Cluster from `3`, which is a minimum recommended value, to `5` nodes.
 
 **Note:** *Using ```kubectl scale StatefulSet_name``` command to rescale Percona XtraDB Cluster is not recommended, as it makes `size` configuration option out of sync, and the next config change may result in reverting the previous number of nodes.*
+
```

Comment:
I think this should have a subheading, as should the preceding selection to separate the acts of scaling instance count in the cluster and scaling volume size for existing instances in the cluster.